src/main/scala/org/scalameter/reporting/HtmlReporter.scala

```diff
@@ -22,7 +22,8 @@ case class HtmlReporter[T: Numeric](embedDsv: Boolean = true) extends Reporter[T
       new DsvReporter(dsvDelimiter).report(results, persistor)
     }
 
-    val resultdir = results.context(reports.resultDir)
+    // resultDir is global setting
+    val resultdir = currentContext(reports.resultDir)
     val root = new File(resultdir, "report")
```

Comment:
It seems this will also cause problem if user sets resultDir from code. It seems the best solution is to remove `resultDir` from `Context`, as discussed from #84:

> The resultDir is a special configuration variable that can only be set through the command line. We do not allow setting different resultDir for different test scopes.